### DISCUSSION TITLE
Serve static assets with an efficient cache policy - Lightshouse diagnostics/NextJS

### DISCUSSION THREAD

**Discussion by tomgreeEn** [vercel/vercel#5191] [Help] [answered]
Hi
In a lighthouse audit of my nextJs site hosted on Vercel, I the warning: Serve static assets with an efficient cache policy - and a list of files with no Cache TTL.

Is this expected behaviour for nextjs/vercel or is my project misconfigured?

Thanks

Tom

![Screenshot 2020-09-14 at 07 29 37](https://user-images.githubusercontent.com/4262865/93051961-37fde900-f65d-11ea-8068-73bb33f23cca.png)

**Comment by alexfoxy** [comment]
@tomgreeEn any success working this out?

**Reply by tomgreeEn** [reply]
Afraid not - I've not looked at it in a while and not yet updated to v10

**Comment by felixmeziere** [comment]
Same problem here with a 1mn cache ttl that I can't modify!

**Comment by conioX** [comment]
any news on this?

**Comment by lhguerra** [comment]
I just discovered that Google/Lighthouse, in their "Serve static assets with an efficient cache policy" test, requires very long max age times (like six months) to give you the best score. But vercel limits it at 31 days, so next/image components always show up as performance improvements in this part of the test.

**Comment by amyegan** [comment, marked as answer]
I usually find Google's documentation helpful for understanding Lighthouse results. You can read more about the "Serve static assets with an efficient cache policy" diagnostic at [web.dev/uses-long-cache-ttl](https://web.dev/uses-long-cache-ttl/). 

> In general, only [metrics](https://web.dev/lighthouse-performance/#metrics) contribute to your Lighthouse Performance score, not the results of Opportunities or Diagnostics. That said, improving the opportunities and diagnostics likely improve the metric values, so there is an indirect relationship.
> —  [Lighthouse performance scoring](https://web.dev/performance-scoring/)

Appropriate cache settings will depend on your site, but there are some more resources that might help:
- [How to Configure the Cache-Control Response Header in Vercel Projects](https://vercel.com/guides/how-to-configure-the-cache-control-response-header-in-vercel-projects)
- Edge Network [Frequently Asked Questions](https://vercel.com/docs/concepts/edge-network/frequently-asked-questions#)
- [Caching responses from Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions/edge-caching#)
- [Incremental Static Regeneration](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration)

**Comment by Mayur290** [comment]
I am using next 11, Having the above issue,

**Comment by shardbearer-shri** [comment]
Can anyone update on this? Getting the same issue, I have also done all optimizations from my end.

**Comment by in-ch** [comment]
Is there any updated information regarding this issue?
